rise-data.html

```diff
@@ -105,6 +111,83 @@
 
         // log usage
         this.$.logger.log(BQ_TABLE_NAME, params);
+      },
+
+      /**
+       * Cache the data
+       *
+       * @param {String} key The key to identify the item of data
+       * @param {Object} data The data to cache
+       * @param {Boolean} ts Optionally include a timestamp to cache
+       */
+      saveItem: function(key, data, ts) {
+        var cacheObj = {};
+
+        if (this._pingReceived && key && data) {
+          if (!this._isCacheRunning) {
+            if (supportsLocalStorage()) {
+              cacheObj.data = data;
+
+              if (ts) {
+                cacheObj.timestamp = moment().format();
+              }
+
+              try {
+                localStorage.setItem(key, JSON.stringify(cacheObj));
+              } catch(e) {
+                console.warn(e.message);
```

Comment:
@stulees Do we have to log it? Can we just ignore it? If we really care about it, we can use `rise-logger`. I don't think we should clutter the console.